src/components/Comment/Comment.vue

```diff
@@ -0,0 +1,211 @@
+              <b> Reply</b>
+            </button>
+            <button
+              v-show="level>=2 && comment.comments.length !== 0"
+              class="comment-button"
+              @click="levelControl = !levelControl"
+            >
+              <i class="bi bi-chat-square"></i>
+              {{ comment.comments.length }} Comment
+            </button>
+            <div v-show="replyIsClick === true" id="share">
+              <vue-simplemde
+                v-model="text"
+                ref="markdownEditor"
+                id="vue-simplemde"
+              />
+              <input type="checkbox" v-model="isSpoiler" />
+              <label>Is spoiler</label>
+              <button @click="share">Share</button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div
+        v-show="comment.comments.length !== 0 && (level<2 || levelControl)"
+        class="nested-comments"
+        v-for="nestedComment in comment.comments"
+        :key="nestedComment.id"
+      >
+        <Comment :level = "level+1" :comment="nestedComment" @share="emitNestedComment" />
+      </div>
+    </div>
+      <div class="spoiler"
+      v-show="comment.is_spoiler === true && isSpoilerClick === false">
+        <button @click="isSpoilerClick = true">View spoiler</button>
+      </div>
+  </div>
+</template>
+<script>
+import axios from 'axios';
+import swal from 'sweetalert';
+import VueSimplemde from 'vue-simplemde';
+
+export default {
+  name: 'Comment',
+  props: {
+    comment: Object,
+    level: Number,
+  },
+  components: {
+    VueSimplemde,
+  },
+  data() {
+    return {
+      levelControl: false,
+      replyIsClick: false,
+      text: null,
+      isSpoiler: false,
+      commentText: null,
+      isSpoilerClick: false,
+    };
+  },
+  methods: {
+    emitNestedComment() {
+      this.$emit('shareComment');
+    },
+    share() {
+      axios
+        .post(
+          '/api/comments/',
+          {
+            text: this.text,
+            parent_comment_id: this.comment.id,
+            is_spoiler: this.isSpoiler,
+          },
+          {
+            headers: {
+              'x-access-token': window.localStorage.getItem('x-access-token'),
+            },
+          },
+        )
+        .then((res) => {
+          if (res.status === 200) {
+            this.replyIsClick = false;
+            this.$emit('share');
+            swal({
+              icon: 'success',
+              text: 'comment created successfully',
+            });
+          }
+        })
+        .catch((err) => {
+          const message = err.response.data.errors.map((e) => e.message);
+          swal({
+            icon: 'error',
+            text: `${message}`,
+          });
+        });
+    },
+  },
+  computed: {
+    simplemde() {
+      return this.$refs.markdownEditor.simplemde;
+    },
+  },
+  mounted() {
+    this.commentText = this.simplemde.markdown(this.comment.text);
+  },
+};
+</script>
+<style lang="scss">
```

Comment:
```suggestion
<style lang="scss" scoped>
```
CSS'in sadece bu sayfada kalması için `scoped` eklenmeli